docs/source/developers/java/development.rst

```diff
@@ -110,9 +110,66 @@ integration tests, you would do:
 Code Style
 ==========
 
-Code style is enforced with Checkstyle. The configuration is located at `checkstyle`_.
+The current Java code styles are configured as follows:
+
+- Indent: Tabs & spaces (2 spaces per tab)
+- Google Java Format: Reformats Java source code to comply with `Google Java Style`_.
+- Configure license headers for Java & XML files
+
+Java code style is checked by `Spotless`_ during the build, and the continuous integration build will verify
+that changes adhere to the style guide.
+
+.. code-block:: xml
+
+    <java>
+      <indent>
+        ...
+      </indent>
+      <googleJavaFormat/>
+      <licenseHeader>
+        ...
+      </licenseHeader>
+    </java>
+    <pom>
+      <indent>
+        ...
+      </indent>
+      <sortPom>
+        ...
+      </sortPom>
+    </pom>
+
+Automatically fixing code style issues
+--------------------------------------
+
+- You can also just check the style without building the project with `mvn spotless:check`.
+- The Java code style can be corrected from the command line by using the following commands: `mvn spotless:apply`.
+
+.. code-block:: bash
+
+    user@machine repo % mvn spotless:check
```

Comment:
Was this copied from their docs? Can we just provide our own example or omit this?

Reply:
I adopted some of the things from the older PR. Let me verify. Still an early draft. I will clean up before marking this ready for review. Sorry about this.